content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,81 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
+The Kubernetes API is a resource-based (RESTful) programmatic interface
+provided via HTTP. It supports retrieving, creating, updating, and deleting
+primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE,
+GET).
 
+The API includes additional subresources for many objects that allow
+fine grained authorization (such as a separating viewing details for a Pod from
+retrieving its logs), and can accept and serve those resources in different
+representations for convenience or efficiency.
+
+Kubernetes supports efficient change notifications on resources via *watches*.
+Kubernetes also provides consistent list operations so that API clients can
+effectively cache, track, and synchronize the state of resources.
+
+You can view the [API reference](/docs/reference/kubernetes-api/) online,
+or read on to learn about the API in general.
 
 <!-- body -->
-The Kubernetes API is a resource-based (RESTful) programmatic interface provided via HTTP. It supports retrieving, creating,
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages standard RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation in JSON (their object schema) which is called a *kind*
+* A list of instances of a resource type is known as a *collection*
```

Comment:
```suggestion
* A list of instances of a resource is known as a *collection*
```

Reply:
I think the original wording is more accurate. If I have a resource `/api/v1/namespaces/default/services/web` then I don't want to list all the instances of that specific web service (the resource); I want to list all the services in the namespace.

Reply:
"resource" doesn't mean a particular object to me. Fortunately I don't have time for another pass so I'll have to be happy with whatever you come up with :)

Reply:
We don't have a good term that gives us a way to differentiate “Deployment” (the generic feature of Kubernetes that lets you manage ReplicaSets) from a specific resource (in the HTTP sense of the term) representing a specific Deployment in a named namespace.